### ISSUE TITLE
2nd time you pinch on mobile it breaks

### ISSUE THREAD

**Issue by stevemarksd** [davidfig/pixi-viewport#200]
Reproduce:

 - Open this page https://davidfig.github.io/pixi-viewport/ in chrome on your android phone
 - Pinch once (all fine)
 - Pinch again

What happens:

 - It kind of freaks out in a strange way

What should happen:
 
 - Same as on first pinch, smooth zoom

**Comment by davidfig** [comment]
I was able to replicate the bug. It's weird because it only happens on the second pinch. Future pinches and all drags work fine.

This is likely caused by pixi's hitArea test for the viewport. The viewport is not getting the pointerdown event for the finger that is outside the viewport during that second pinch. Weirdly, it does get pointermove events when that finger moves, but it's getting the same id as the other finger, which is why the viewport gets wonky: it's performing the drag calculation using two separate finger locations.

Is this causing problems in your app? I'd have to dig deeper and figure out where the events are getting crossed.

**Comment by stevemarksd** [comment]
yes it is causing :( everything works quite smoothly. there is actually another bug and I will open now

**Comment by moljac024** [comment]
This seems to fixed in the latest release as of right now.

**Comment by devyeshtandon** [comment]
Were we able to figure out the root cause of this bug?
